Authenticator/Source/QRScanner.swift

```diff
@@ -31,7 +31,7 @@ protocol QRScannerDelegate: class {
 
 class QRScanner: NSObject, AVCaptureMetadataOutputObjectsDelegate {
     weak var delegate: QRScannerDelegate?
-    private let serialQueue = dispatch_queue_create("QR Scanner serial queue", DISPATCH_QUEUE_SERIAL);
+    private let serialQueue = dispatch_queue_create("QR Scanner serial queue", DISPATCH_QUEUE_SERIAL)
```

Comment:
Length Violation: Line should be 100 characters or less: currently 101 characters